### PR TITLE
Add calc() to @media.

### PR DESCRIPTION
**Pull request by jpmedley**
https://storage.googleapis.com/chromium-find-releases-static/5e7.html#5e7c6952c431e507a56c7fbd9484571ff81ea4e3

### CODE REVIEW

File: css/at-rules/media.json [hunk 1]
```diff
@@ -307,6 +307,58 @@
             }
           }
         },
+        "calc": {
+          "__compat": {
+            "description": "<code>calc()</code> expressions",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/color",
```
Comment (ddbeck) [inline css/at-rules/media.json]:
```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/calc",
```

**Comment by ExE-Boss** [comment]
This would benefit from #1799.

**Comment by Elchi3** [comment]
Hm, "css.at-rules.media.calc" does make sense to me. I wonder if we should rather remove "css.types.calc.media_query_values". To me, it is rather a sub feature of media queries than a sub feature of the calc type.

**Comment by ddbeck** [comment]
OK, that sounds reasonable to me. Thanks for your input, @Elchi3.

@jpmedley would you mind transfering values from `css.types.calc.media_query_values` to your new `css.at-rules.media.calc` feature and then removing `css.types.calc.media_query_values`?

**Comment by jpmedley** [comment]
I've done the move. Please note that the move and a subsequent alphabetization are in two separate commits so that you can easily see what has been done.

File: css/at-rules/media.json (hunk 1 above)
Comment (gibriskateboy) [reply css/at-rules/media.json]: https://www.google.com/url?hl=en-GB&q=https://github.com/mdn/browser-compat-data/pull/3549&source=gmail&ust=1551813036700000&usg=AFQjCNGIWradyaXx_VujiOFTWnrh7rDFUQ

File: css/at-rules/media.json [hunk 2]
```diff
@@ -310,7 +310,7 @@
         "calc": {
           "__compat": {
             "description": "<code>calc()</code> expressions",
-            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/color",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/calc",
```
Comment (Satansseed6) [inline css/at-rules/media.json]: Correct

**Pull request merged**